### ISSUE TITLE
Grid - No keyboard navigation available

### ISSUE THREAD

**Issue by justinrassier** [telerik/kendo-angular#897]
## I'm submitting...
a feature request -

I have submitted an item to the feedback page, but have not heard about any follow-up. http://kendoui-feedback.telerik.com/forums/555517-kendo-ui-for-angular-feedback/suggestions/20458870-keyboard-navigations-for-grid

## Current behavior
The new Kendo grid for Angular does not have any keyboard navigation available

## Expected behavior
Many Kendo components, as well as the old grid, have keyboard navigation features, I think the grid should also have more advanced keyboard navigation included.

## Minimal reproduction of the problem with instructions
N/A

## What is the motivation or use case for changing the behavior?
The grid should meet a11y minimums.

## Environment

Package versions:
Latest

Browser:
N/A

System:
N/A

**Comment by tsvetomir** [comment]
The keyboard navigation and accessibility indeed got delayed. We're reviving our efforts there and both will be shipped with the next major release.

Apologies for the delay. We'll update the UserVoice item and will close this issue to avoid duplication.
